lms/templates/help_modal.html

```diff
@@ -208,7 +202,7 @@
     cycle_modal_tab("#feedback_submit", "#feedback_form_wrapper .close-modal");
     cycle_modal_tab("#feedback-faq-link", "#feedback_success_wrapper .close-modal");
 % else:
-    cycle_modal_tab("#howto-enroll-link", "#help_wrapper .close-modal");
+    cycle_modal_tab("#last-link", "#help_wrapper .close-modal");
```

Comment:
if cycle_modal_can handle a class, you should DEFINITELY change it to a class...what if at some point we want to have to tabbed modals available on a page?